### PR TITLE
Fixed an issue in IE10 and below

### PR DESCRIPTION
**Pull request by spassvogel**
Where it would throw 'redeclaration of const' error

Should fix https://github.com/bodymovin/bodymovin/issues/542

### CODE REVIEW

File: player/js/module.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-var window = (typeof window === "undefined") ? {} : window;
+if(typeof window === "undefined") { 
+    window = {}; 
+}
```
Comment (piotar) [inline player/js/module.js]: Your solution will work if variable `window`  has value but, if variable value is `undefined` you still want redeclarate const variable.

You should forward variable `window` or new object as argument.

I think this is better solution:
```
(function(root, factory) {
    if (typeof define === "function" && define.amd) {
        define(function() {
            return factory(root);
        });
    } else if (typeof module === "object" && module.exports) {
        module.exports = factory(root);
    } else {
        root.bodymovin = factory(root);
    }
}((window || {}), function(window) {
    /*<%= contents %>*/
    var bodymovinjs = {};
...
```

**Comment by piotar** [comment]
It will fix bug #596 too.

File: player/js/module.js (hunk 1 above)
Comment (spassvogel) [reply player/js/module.js]: Hi! Thanks for checking this PR, however your solution causes a runtime error on 

`TypeError: window is undefined  bodymovin.js:1143:42`

This seems to be coming from `animationFramePolyFill`: https://github.com/spassvogel/bodymovin/blob/d73e555bb0502d015e342bb3fec8777d9b0b7c66/player/js/utils/animationFramePolyFill.js

Comment (piotar) [reply player/js/module.js]: Hm, it weird, my solution work for me. (checked in IE9 and IE10)

Maybe you must look at all code what I wrote before, not only line: 
`}(window, function() {`

Here you can look what I changed: https://github.com/piotar/bodymovin/commit/29442bc8e8d2b6d1a28d499cdbb35bc3eca5b5e1#diff-828198e63658bf16405f5deb7fc7be05

`TypeError: window is undefined bodymovin.js:1143:42` 
this error shown when we not pass parameter `window`(as root) in factory
```
 if (typeof define === "function" && define.amd) {
        define(function() {
            return factory(root);
        });
    } else if (typeof module === "object" && module.exports) {
        module.exports = factory(root);
    } else {
        root.bodymovin = factory(root);
    }
```

Comment (spassvogel) [reply player/js/module.js]: Apologies you are right I did not copy your code completely. Maybe you should make a Pull Request then I will withdraw mine.

**Comment by bodymovin** [comment]
Does this PR fix the issue with ie9 and 10?

File: player/js/module.js (hunk 1 above)
Comment (piotar) [reply player/js/module.js]: Please fill by yourself, to quick fix this problem ;)

**Comment by piotar** [comment]
Yes, this PR will fix bug on IE9 and IE10 problem with redeclaration of const property.

**Comment by spassvogel** [comment]
Hang on let me implement @piotar s changes!

**Comment by spassvogel** [comment]
Okay it's working now thanks to @piotar. Updated the PR with c9832ba.

**Comment by spassvogel** [comment]
Is this repo still maintained?

**Comment by bodymovin** [comment]
@spassvogel it is. Sorry about the delay.

**Pull request merged**